[CC-14560] Enable prioritization of discounts (#556)
spryker/spryker-docs: docs/scos/user/back-office-user-guides/202108.0/merchandising/discount/creating-cart-rules.md

To create a cart rule discount:
1. On the *Discount* page, in the top right corner, click **Create new discount**.
2. On the *Create Discount* page, in the *General information* tab, do the following:
    1. In *STORE RELATION*, check the stores you wish the discount to be active in.
    2. From the *DISCOUNT TYPE* drop-down list, select a cart rule.
    3. In the *NAME* field, specify the name for the discount.
    4. _Optional_: in the *DESCRIPTION* field, enter the description of the discount.

>     5. _Optional_: in the *PRIORITY* field, enter an integer value from 1 to 9999 for the discount priority. For reference information, in the Creating a voucher guide, see the [Discount Overview page](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-overview-page) section.

Author: I've updated the text here https://github.com/spryker/spryker-docs/pull/556/files#diff-0bcb81979bb89cfd63bd0c17b8de93731fd5225062715d82d5172f1b1974fef1R35, there was a mistake

    6. Specify if the discount is exclusive. For reference information, in the *Create new discount* reference, see [General information tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#general-information-tab).
    7. Specify the validity interval (lifetime) of the discount.
* Click **Next** or select the *Discount calculation* tab to proceed.

* On the *Create Discount* page, in the *Discount calculation* tab, do the following:
    1.  In the *CALCULATOR TYPE* drop-down, select either *Percentage* or *Fixed amount*. For reference information, in the *Create new discount* reference, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.

        {% info_block warningBox "Note" %}

        The next step varies based on the selected calculator type:

          a. *Fixed amount*: Enter the amounts (per currency and price mode, if applicable) to be discounted.
          
          b. *Percentage*: Enter the percent value to be discounted.

        {% endinfo_block %}

>     2. Select the *DISCOUNT APPLICATION TYPE* and define what products the discount will be applied to. For reference information, in the *Create new discount* reference, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.

Reviewer: ```suggestion
    2. Select the *DISCOUNT APPLICATION TYPE* and define what products the discount is applied to. For reference information, in the *Create new discount* reference, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
```
